src/components/web3/ConnectButton.tsx

```diff
@@ -1,17 +1,55 @@
+import { WalletOutlined } from '@ant-design/icons'
 import { useWeb3React } from '@web3-react/core'
 import { Button } from 'antd'
-import React from 'react'
 
-import { getInjectedConnector } from './connectors'
+import {
+  readDeactivatedWallets,
+  readWallets,
+  storeDeactivatedWallets,
+  storeWallets,
+} from '../../services/localStorage'
+import { getInjectedConnector, injected } from './connectors'
 
-function ConnectButton() {
+const addToActiveWallets = (address: string | null | undefined) => {
+  if (!address) return
+  const lowerCaseAddress = address.toLowerCase()
+  const activeWallets = readWallets()
+  activeWallets.push(lowerCaseAddress)
+  storeWallets(activeWallets)
+}
+
+const removeFromDeativatedWallets = (address: string | null | undefined) => {
```

Comment:
```suggestion
const removeFromDeactivatedWallets = (address: string | null | undefined) => {
```

Webstorm has a really great spell checker... even validates grammar.... just sayin'....